packages/ui-library/src/components/transaction-receipt/index.tsx

```diff
@@ -323,13 +323,13 @@ const ERC20ApprovalTransactionReceipt = ({ transaction }: { transaction: ERC20Ap
         </Typography>
       </StyledSectionContent>
       <StyledSectionContent>
-        <Typography variant="bodySmallLabel">
+        <Typography variant="labelRegular">
           <FormattedMessage description="TransactionReceipt-transactionAmountSent" defaultMessage="From address" />
         </Typography>
         <StyledBodySmallBold>{transaction.data.owner}</StyledBodySmallBold>
       </StyledSectionContent>
       <StyledSectionContent>
-        <Typography variant="bodySmallLabel">
+        <Typography variant="labelRegular">
           <FormattedMessage description="TransactionReceipt-transactionAmountSent" defaultMessage="Spender" />
         </Typography>
         <StyledBodySmallBold>{transaction.data.spender}</StyledBodySmallBold>
```

Comment:
same as for most of StyledBodySmallBold items

Reply:
Ohhh I see figma's saying Body/Bold now. But still I think thats incorrect and it should be body small (i'll ping the designers for consistency)